stix_shifter_modules/guardium/stix_translation/json/stix_2_1/to_stix_map.json

```diff
@@ -0,0 +1,256 @@
+    },
+    {
+      "key": "ipv6-addr.value",
+      "object": "dst_ip"
+    },
+    {
+      "key": "x-ibm-finding.dst_ip_ref",
+      "object": "finding", 
+      "references": "dst_ip" 
+    }
+  ],
+  "db_type":[
+	{
+	"key":"software.name",
+	"object":"software"
+	},
+	{
+	"key":"x-ibm-finding.dst_application_ref",
+	"object":"finding",
+	"references":"software"
+	}
+  ],
+ "Server Type":[
+	{
+	"key":"software.name",
+	"object":"software"
+	},
+	{
+	"key":"x-ibm-finding.dst_application_ref",
+	"object":"finding",
+	"references":"software"
+	}
+  ],
+  "source_program":[
+	{
+	"key":"software.name",
+	"object":"source_software"
+	},
+	{
+	"key":"x-ibm-finding.src_application_ref",
+	"object":"finding",
+	"references":"source_software"
+	}
+  ],
+  "EventCount": {
+    "key": "number_observed",
+    "cybox": false,
+    "transformer": "ToInteger"
+  },
+  "timestamp": [
+    {
+      "key": "first_observed",
+      "transformer": "GuardiumToTimestamp",
+      "cybox": false
+     },
+    {
+      "key": "last_observed",
+      "transformer": "GuardiumToTimestamp",
+      "cybox": false
+    }
+  ],
+  "Create Date": [
+    {
+      "key": "first_observed",
+      "transformer": "GuardiumToTimestamp",
+      "cybox": false
+    }
+  ],
+  "Timestamp": [
+    {
+      "key": "last_observed",
+      "transformer": "GuardiumToTimestamp",
+      "cybox": false
+    }
+  ],
+  "Last updated": [
+    {
+      "key": "last_observed",
+      "transformer": "GuardiumToTimestamp",
+      "cybox": false
+    }
+  ],
+  "Url": {
+    "key": "url.value"
+  },
+  "FileName": {
+    "key": "file.name"
+  },
+  "Payload": {
+    "key": "artifact.payload_bin"
+  },
+  "DestinationPort": {
+    "key": "network-traffic.dst_port",
+    "object": "nt",
+    "transformer": "ToInteger"
+  },
+  "SourcePort": {
+    "key": "network-traffic.src_port",
+    "object": "nt",
+    "transformer": "ToInteger"
+  },
+  "NetworkProtocol": {
+    "key": "network-traffic.protocols",
+    "object": "nt",
+    "transformer": "ToLowercaseArray"
+  },
+  "DomainName": {
+    "key": "domain-name.value"
+  },
+  "Process": {
+    "Path": [
+      {
+        "object": "proc",
+        "key": "process.command_line"
+      }
+    ],
+    "Pid": [
+      {
+        "object": "proc",
+        "key": "process.pid"
+      }
+    ]
+  },
+  "violation":{
+	"key":"x-ibm-finding.name",
+        "object":"finding"
+  },
+ "Threat Category":{
+	"key":"x-ibm-finding.name",
+        "object":"finding"
+  },
+  "details":{
+	"key":"x-ibm-finding.details",
+        "object":"finding"
+  },
+ "database":{
+	"key":"x-ibm-finding.dst_device",
+        "object":"finding"
+  },
+ "Database":{
+	"key":"x-ibm-finding.dst_device",
+        "object":"finding"
+  },
+  "severity":[
+     {
+	"key":"x-guardium.severity",
+     "transformer": "GuardiumMapSeverity"},
+     {
+    "key":"x-ibm-finding.finding_type",
+	"object":"finding",
+    "transformer": "GuardiumQS"}
+  ],
+  "Severity":[
```

Comment:
Why is there both `severity` and `Severity` coming in? Are these two fields that Guardium returns? Also, why isn't this written to the `x-ibm-finding:severity` property? How does this value come in? The finding object expects it to be on a scale of 1-100. See the finding definition at https://github.com/opencybersecurityalliance/stix-extensions/blob/main/2.0/x-ibm-finding.md.

Reply:
Guardium api returns two different severity one is for report and another one is coming from quick search.